### DISCUSSION TITLE
"Name or service not known" errors should include hostname or url

### DISCUSSION THREAD

**Discussion by trondhindenes** [encode/httpx#3008] [Ideas]
From time to time we're having some dns-related issues in our infrastructure, so the ""Name or service not known" exceptions are very important to us. We're noticing that when httpx raises this exception, there's no way to see the url or hostname it actually tried to resolve. It would be very valuable if that was surfaced

**Comment by lovelydinosaur** [comment]
Yep, agreed. If you'd be up for making a contribution to `httpcore` could point you in the right direction?

**Reply by trondhindenes** [reply]
absolutely. Let me poke around a bit on my own during the holidays, will ask in this thread if I need help. Thanks for replying!
